FabLabKasse/UI/ClientDialogCode.py

```diff
@@ -173,3 +185,81 @@ def reject(self):
         msgBox.setText(u"Bezahlung abgebrochen.")
         msgBox.exec_()
         QtGui.QDialog.reject(self)
+    
+    def ask_admin_pin(self):
+        cfg = scriptHelper.getConfig()
+        try:
+            pin = cfg.get('payup_methods', 'client_admin_pin');
+        except ConfigParserError:
+            msgBox = QtGui.QMessageBox(self)
+            msgBox.setText(u"fehlende Konfiguration")
+            msgBox.exec_()
+            return False
+        admin_pin = KeyboardDialog.askText("Admin PIN?", parent=self, hidden_input=True)
```

Comment:
```suggestion
        admin_pin = KeyboardDialog.askText(u"Admin PIN?", parent=self, hidden_input=True)
```
Is it intentional not to use a unicode literal here?